build_ffmpeg.sh

```diff
@@ -32,22 +32,22 @@ if [ "$1" = "--init" ]; then
 
     printf "NDK\n"
 
-    if [ ! -f android-ndk32-r10-darwin-x86_64.tar.bz2 ]; then
-        printf "    downloading\n"
-        curl "http://dl.google.com/android/ndk/android-ndk32-r10-darwin-x86_64.tar.bz2" \
-            -o android-ndk32-r10-darwin-x86_64.tar.bz2 \
+    if [ ! -f android-ndk-r12b-darwin-x86_64.zip ]; then
+        printf "    downloading r12b for mac\n"
+        curl "https://dl.google.com/android/repository/android-ndk-r12b-darwin-x86_64.zip" \
+            -o android-ndk-r12b-darwin-x86_64.zip \
             >> ${LOG_FILE} 2>&1
     fi
 
-    printf "    extracting\n"
-    tar xvzf android-ndk32-r10-darwin-x86_64.tar.bz2 \
+    printf "    unzipping\n"
+    unzip android-ndk-r12b-darwin-x86_64.zip \
```

Comment:
nit: Could make ndk-r12b into a constant so it's easy to update in the future. Feel free to merge without this change.